client/fixtures/productDetail.ts

```diff
@@ -687,3 +694,16 @@ export const cancelledContribution: ProductDetail = {
 	},
 	isTestUser: false,
 };
+
+export const membersDataApiResponse = (
```

Comment:
This is a transformation function, yes? Might read better as `toMembersDataApiResponse` ?